tests/test_plugin.py

```diff
@@ -237,6 +237,27 @@ def test_is_testrun_available(api_client, tr_plugin):
     assert tr_plugin.is_testrun_available() is False
 
 
+@pytest.mark.parametrize('include_all', [True, False])
```

Comment:
You should test at a higher level by calling `pytest_collection_modifyitems`.

You can see an example at [`test_skip_missing_only_one_test`](https://github.com/allankp/pytest-testrail/blob/b2003fa223e36501473c1184a0df37f90951155d/tests/test_plugin.py#L340)